internal/controller/api/v1alpha1/agentserviceconfig_types.go

```diff
@@ -26,10 +26,15 @@ import (
 type AgentServiceConfigSpec struct {
 	// FileSystemStorage defines the spec of the PersistentVolumeClaim to be
 	// created for the assisted-service's filesystem (logs, etc).
+	// With respect to the resource requests, the amount of filesystem storage
+	// consumer will depend largely on the number of clusters you intend to create.
+	// It's best to request as much as possible with the minimum being 20GiB in
+	// development/testing and 100Gib anywhere else.
```

Comment:
I got these numbers from my testing environment, which may help with providing a more accurate indication: 

* ~110 MB per cluster (this includes ignition, install-config, and other files as well as the minimal-iso)
* ~2.2 GB per supported version (see all the rhcos files `rhcos-47.83.202102090044-0-minimal.iso  rhcos-47.83.202102090044-0.initrd.img   rhcos-47.83.202102090044-0.iso          rhcos-47.83.202102090044-0.rootfs.img   rhcos-47.83.202102090044-0.vmlinuz` )


Here's the screenshot

![DeepinScreenshot_select-area_20210409130338](https://user-images.githubusercontent.com/13816/114177677-9a3d7480-9934-11eb-9340-1485fecf6006.png)